### PR TITLE
 fixed the issue #4660 - shouldn't use a hook like function name for a variable

### PR DESCRIPTION
**Pull request by Kaku-g**
changed the name of the variable from hook like function name  to normal variable name.

### CODE REVIEW

**Comment by sneridagh** [comment]
@Kaku-g thanks for the contribution, but I don't think it's relevant issue for now.

**Comment by davisagli** [comment]
@sneridagh It was in response to an issue Tiberiu opened -- should we close #4660 as well?

**Comment by sneridagh** [comment]
@davisagli Uh, I missed that Tiberiu was asking for it... in that case, let's consider it again.

**Review by sneridagh — CHANGES_REQUESTED** [review]
@Kaku-g It needs a Towncrier changelog entry too. Thanks!

**Comment by Kaku-g** [comment]
@sneridagh done the requested changes!

File: packages/volto-slate/src/blocks/Text/SlashMenu.jsx [hunk 1]
```diff
@@ -110,7 +110,7 @@ const PersistentSlashMenu = ({ editor }) => {
 
   const [slashMenuSelected, setSlashMenuSelected] = React.useState(0);
 
-  const useAllowedBlocks = !isEmpty(allowedBlocks);
+  const allowedBlockSection  = !isEmpty(allowedBlocks);
```
[the remark this answers is not in the document]
Comment (Kaku-g) [reply packages/volto-slate/src/blocks/Text/SlashMenu.jsx]: Updated the required changes

**Review by sneridagh — APPROVED** [review]

**Pull request merged**